pkg/model/catalog.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package model
```

Comment:
This is very good suggestions. I found some examples https://github.com/cloudfoundry-incubator/cf-swagger I will look into it. Since it will take me some time, I would prefer loading this CL in first and iterate over.